src/collector/components/component.ts

```diff
@@ -27,9 +28,43 @@ export interface IComponent<M extends IFile = IFile, W extends IFile = IFile, S
     }
 }
 
+function extractCorrectTwigFileName(defaultFileName: string, directoryFiles: string[]) {
+    const twigFileNameInComponent = directoryFiles.filter(fileName => fileName.indexOf('.twig') >= 0)[0];
+    if (!directoryFiles.includes(defaultFileName) && twigFileNameInComponent) {
+        return twigFileNameInComponent;
+    }
+
+    return defaultFileName;
+}
+
+function extractCorrectScssFileName(defaultFileName: string, directoryFiles: string[]) {
+    const scssFileNameInComponent = directoryFiles.filter(fileName =>
+        fileName.indexOf('.scss') >= 0 && fileName !== 'style.scss'
```

Comment:
Suggest to replace fileName.indexOf('. scss') >= 0 -> fileName.includes('. scss')